licenses/LICENSE-xz.txt

```diff
@@ -0,0 +1,11 @@
+Permission to use, copy, modify, and/or distribute this
```

Comment:
Hi @dongjoon-hyun

This file should be placed in the licenses-binary folder as the `xz` belongs to the binary releases

Reply:
Thank you. Let me make a follow-up, @yaooqinn .

Reply:
Thank you, @yaooqinn .
Here is the follow-up.
- https://github.com/apache/spark/pull/48107